src/pystatis/config.py

```diff
@@ -26,6 +26,9 @@
 PKG_NAME = __name__.split(".", maxsplit=1)[0]
 DEFAULT_CONFIG_DIR = str(Path().home() / f".{PKG_NAME}")
 SUPPORTED_DB = ["genesis", "zensus", "regio"]
+REGEX_DB = ["^((\d{5}-\d{4})|([0-9A-Z]{10}))$", 
```

Comment:
I would not use a list here because it is unclear which regex belongs to which database. I would always suggest using a dictionary for that kind of mapping because that is what it is about: mapping a database name to the regex that belongs to it. This is also much more user friendly because you can simply extract the regex by `REGEX_DB.get("genesis")` instead of looking up the index

Reply:
I think it would be better to use the `re` module here instead of the different places, so I actually would like to store a precompiled regex here, like `re.compile(...)`, this will allow you to later just call `REGEX_DB["genesis"].filterall(<name>)`

Reply:
@PiaSchroeder this is still relevant